tensorflow/third_party/mkl/BUILD

```diff
@@ -0,0 +1,9 @@
+# This file is replaced on github with a real build rule.
```

Comment:
Indeed. I kept the ones that reside in third_party/mkl, even though they won't work since the .so files are missing.